3rdParty/CMakeLists.txt

```diff
@@ -101,11 +101,21 @@ target_include_directories(v8_interface SYSTEM INTERFACE ${V8_INCLUDE_DIR})
 
 set(ICU_DT "${ICU_DT}" PARENT_SCOPE)
 set(ICU_DT_DEST "${ICU_DT_DEST}" PARENT_SCOPE)
-set(ICU_LIBRARY_DIR "${ICU_LIBRARY_DIR}" PARENT_SCOPE)
-set(ICU_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/${V8_SUB_DIR}/third_party/icu/")
+#set(ICU_LIBRARY_DIR "${ICU_LIBRARY_DIR}" PARENT_SCOPE)
+#set(ICU_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/${V8_SUB_DIR}/third_party/icu/")
```

Comment:
We can remove these two lines, since they are commented out anyway and the variables are set below properly.
```suggestion
```